src/tools/async-download/components/AsyncDownloadConfirmation.tsx

```diff
@@ -100,11 +105,36 @@ const AsyncDownloadConfirmation = ({
       content: jobParameters.columns.join(', ') || '',
     },
   ];
+
   return (
     <Card
       header={<h4>Review your file generation request</h4>}
       className={styles['confirm-async-download']}
     >
+      {jobParameters.query === '*' ? (
+        <Message className={styles['warning-message']} level="warning">
+          This job will take longer to finish, as your search is against the
+          whole of {namespaceAndToolsLabels[jobParameters.namespace]}, making it
+          a highly resource-intensive process. We kindly request that you
+          download the file once it becomes available.
+        </Message>
+      ) : null}
+      {jobParameters.query !== '*' &&
+      jobParameters.query?.split(' ').length === 1 &&
```

Comment:
I think l.123 might not be needed, the most important is l.124 (checking for the absence of ":"). Search for "homo sapiens" is also a free text search that might not return what the user expected.
Unless I'm missing  a use case that this covers?